fiducial_slam/launch/fiducial_slam.launch

```diff
@@ -10,6 +10,7 @@
   <arg name="future_date_transforms" default="0.0"/>
   <arg name="publish_6dof_pose" default="false"/>
   <arg name="fiducial_len" default="0.14"/>
+  <arg name="systematic_error" default="0.01"/>
```

Comment:
I don't like the name, something like `minimum_system_error` or `minimum_error` might be better.

Reply:
Systematic error is a term of art and has a specific and exact meaning. Specifically systematic error is the error that is system wide and thus non-random. Systematic errors are pernicious and difficult to resolve. Minimum error implies that that this is the lowest error possible - that's not what this is - the minimum error could be significantly larger than the systematic error. Could we please stick with the term "systematic error" or at least use something that describes what it is for example "non-random_error" or similar.

Reply:
https://www.physics.umd.edu/courses/Phys276/Hill/Information/Notes/ErrorAnalysis.html